lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -5,16 +5,24 @@
         'js/learner_dashboard/views/collection_list_view',
         'js/learner_dashboard/views/sidebar_view',
         'js/learner_dashboard/views/program_card_view',
+        'js/learner_dashboard/views/empty_programs_list',
         'js/learner_dashboard/collections/program_collection'
     ],
-    function (CollectionListView, SidebarView, ProgramCardView, ProgramCollection) {
+    function (CollectionListView, SidebarView, ProgramCardView, EmptyProgramsList, ProgramCollection) {
         return function (options) {
-            new CollectionListView({
-                el: '.program-cards-container',
-                childView: ProgramCardView,
-                collection: new ProgramCollection(options.programsData)
-            }).render();
 
+            if (!options.length) {
+                new EmptyProgramsList({
```

Comment:
Could you please follow the naming convention and name it as  `EmptyProgramsView`?